src/data/delegators/entityMetaProvider.tsx

```diff
@@ -1,6 +1,4 @@
 import { DrupalJsonApiParams } from 'drupal-jsonapi-params'
```

Comment:
we may be able to nuke this entirely now?

Reply:
Indeed, I'll open a quick branch to do that.